deploy/crds/app.kiegroup.org_kogitojobsservices_crd.yaml

```diff
@@ -0,0 +1,350 @@
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: KogitoJobsServiceSpec defines the desired state of KogitoJobsService
+          properties:
+            backOffRetryMillis:
+              description: 'Retry backOff time in milliseconds between the job execution
+                attempts, in case of execution failure. Default to service default,
+                see: https://github.com/kiegroup/kogito-runtimes/wiki/Jobs-Service#configuration-properties'
+              format: int64
+              type: integer
+            envs:
+              description: Environment variables to be added to the runtime container.
+                Keys must be a C_IDENTIFIER.
+              items:
+                description: EnvVar represents an environment variable present in
+                  a Container.
+                properties:
+                  name:
+                    description: Name of the environment variable. Must be a C_IDENTIFIER.
+                    type: string
+                  value:
+                    description: 'Variable references $(VAR_NAME) are expanded using
+                      the previous defined environment variables in the container
+                      and any service environment variables. If a variable cannot
+                      be resolved, the reference in the input string will be unchanged.
+                      The $(VAR_NAME) syntax can be escaped with a double $$, ie:
+                      $$(VAR_NAME). Escaped references will never be expanded, regardless
+                      of whether the variable exists or not. Defaults to "".'
+                    type: string
+                  valueFrom:
+                    description: Source for the environment variable's value. Cannot
+                      be used if value is not empty.
+                    properties:
+                      configMapKeyRef:
+                        description: Selects a key of a ConfigMap.
+                        properties:
+                          key:
+                            description: The key to select.
+                            type: string
+                          name:
+                            description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                              TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Is it needed or can be removed?

Reply:
Generator issues, I'll remove it.

Reply:
@sutaakar this is generated by the Kubernetes API, we can't do much about it. It's from the `EnvVar` resource.

Reply:
ok